openedx/core/djangoapps/catalog/management/commands/tests/test_sync_course_runs.py

```diff
@@ -44,12 +45,18 @@ def test_marketing_url_on_sync(self, mock_catalog_course_runs):
         """
         mock_catalog_course_runs.return_value = [self.catalog_course_run]
         earlier_marketing_url = self.get_course_overview_marketing_url(self.course.id)
+        course_overview = CourseOverview.objects.get(id=self.course.id)
+        earlier_eligible_for_financial_aid = course_overview.eligible_for_financial_aid
 
         call_command('sync_course_runs')
+        course_overview.refresh_from_db()
         updated_marketing_url = self.get_course_overview_marketing_url(self.course.id)
+        updated_eligible_for_financial_aid = course_overview.eligible_for_financial_aid
```

Comment:
This test case covers the sync of the marketing url. Either make the name and documentation for the test more general, or write your own, separate test that covers the financial aid eligibility bit. Don't just co-op this test.